modules/fbx/editor_scene_importer_fbx.cpp

```diff
@@ -180,10 +180,12 @@ Node3D *EditorSceneImporterFBX::import_scene(const String &p_path, uint32_t p_fl
 			}
 
 			if (is_blender_fbx) {
-				WARN_PRINT("Blender FBX files will not work properly with keyframes or skeletons until we make fixes. Please stand by.");
+				WARN_PRINT("We officially don't support blender FBX animations yet, due to issues with upstream blender, \n"
+						   "so please wait for us to add this in. We will continue to import the file but it may be broken.\n"
+						   "for minimal breakage please export fbx from blender with -Z forward, and Y up.");
```

Comment:
Same as `3.2` PR:
```suggestion
				WARN_PRINT("We don't officially support Blender FBX animations yet, due to issues with upstream Blender,\n"
						   "so please wait for us to work around remaining issues. We will continue to import the file but it may be broken.\n"
						   "For minimal breakage, please export FBX from Blender with -Z forward, and Y up.");
```